From 9e0ee97de976d840819e2fa27d0baa7a6aa8e7d4 Mon Sep 17 00:00:00 2001
From: Mahesh Bodapati <mbodapat@xilinx.com>
Date: Fri, 28 Oct 2022 13:57:48 +0530
Subject: [PATCH 34/34] By default the linker will generate warnings if it is
 creating an executable stack or a segment with all three of read, write and
 execute permissions. These settings are not appropriate for all targets

---
 ld/configure.tgt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ld/configure.tgt b/ld/configure.tgt
index e3fd0c2452b..6d6db66b99b 100644
--- a/ld/configure.tgt
+++ b/ld/configure.tgt
@@ -55,6 +55,7 @@ if test "${ac_default_ld_warn_rwx_segments}" = unset; then
     cris-*-* | crisv32-*-* | \
     hppa*-*-* | \
     mips*-*-* | \
+    microblaze*-*-* | \
     sparc*-*-* | \
     v850*-*-*)
       ac_default_ld_warn_rwx_segments=0
-- 
2.17.1

